docs/getstarted/userinterface.md

```diff
@@ -53,6 +53,19 @@ When you have more than one editor open you can switch between them quickly by h
 
 >**Tip:** You can resize editors and reorder them. Drag and drop the editor title area to reposition or resize the editor.
 
+## Secondary Side Bar
+
+The Secondary Side Bar can display views opposite the Primary Side Bar. You can open it as follows:
+
+- Use the keyboard shortcut `kb(workbench.action.toggleAuxiliaryBar)`.
+- Open the [Command Palette](/docs/getstarted/userinterface.md#command-palette) with the keyboard shortcut `kb(workbench.action.showCommands)` and run the `View: Toggle Secondary Side Bar Visibility` command.
```

Comment:
The convention is to use bold for commands so **View: Toggle Secondary Side Bar Visibility**

Reply:
```suggestion
- Open the [Command Palette](/docs/getstarted/userinterface.md#command-palette) with the keyboard shortcut `kb(workbench.action.showCommands)` and run the **View: Toggle Secondary Side Bar Visibility** command.
```

Thanks for the hint!